terraform/terraform.tfvars.example

```diff
@@ -0,0 +1,26 @@
+# Required Customization
+#
+aws_key_pair_name = "jmiller"
+aws_key_pair_file = "~/.ssh/jmiller"
+tag_dept = "SCE"
+tag_contact = "jeremy"
+#
+# Optional
+#
+aws_ami_user = "centos"
```

Comment:
Since all of these are optional, I think we should have them commented out by default but reflecting the default values.